modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -438,6 +438,10 @@
 	margin-bottom: 2em;
 }
 
```

Comment:
Hey @naraandrade  , we need to specify more this selector. I mean, we need to use some Calendar specific identifier to avoid changing/interfering with the style from other portlets.

Thx